test/testdata/lsp/symbol_query_filter__helper.rb

```diff
@@ -0,0 +1,51 @@
+# typed: true
```

Comment:
Could we add tests for aliases?

Reply:
@DarkDimius does this mean "Find All References" for `B` should return all references to `A` and `B`?

If so, then is there an easy way to retrieve all type aliases for a given symbol?

Reply:
IRL; mainly, we need a test to define what behavior to use.